target/linux/generic/files/drivers/net/phy/rtl8367b.c

```diff
@@ -688,18 +713,24 @@ static int rtl8367b_write_phy_reg(struct rtl8366_smi *smi,
 static int rtl8367b_init_regs(struct rtl8366_smi *smi)
 {
 	const struct rtl8367b_initval *initvals;
+	u32 chip_num;
 	u32 chip_ver;
 	u32 rlvid;
 	int count;
 	int err;
 
 	REG_WR(smi, RTL8367B_RTL_MAGIC_ID_REG, RTL8367B_RTL_MAGIC_ID_VAL);
+	REG_RD(smi, RTL8367B_CHIP_NUMBER_REG, &chip_num);
 	REG_RD(smi, RTL8367B_CHIP_VER_REG, &chip_ver);
 
 	rlvid = (chip_ver >> RTL8367B_CHIP_VER_RLVID_SHIFT) &
 		RTL8367B_CHIP_VER_RLVID_MASK;
 
-	switch (rlvid) {
+	dev_info("checking chip_num=0x%x ver=0x%x...\n", chip_num, chip_ver);
```

Comment:
I gave a try and it broke here.

```
drivers/net/phy/rtl8367b.c: In function 'rtl8367b_init_regs':
drivers/net/phy/rtl8367b.c:729:11: error: passing argument 1 of '_dev_info' from incompatible pointer type [-Werror=incompatible-pointer-types]
  dev_info("checking chip_num=0x%x ver=0x%x...\n", chip_num, chip_ver);
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
./include/linux/device.h:1346:46: note: in definition of macro 'dev_info'
 #define dev_info(dev, fmt, arg...) _dev_info(dev, fmt, ##arg)
                                              ^~~
./include/linux/device.h:1295:37: note: expected 'const struct device *' but argument is of type 'char *'
 void _dev_info(const struct device *dev, const char *fmt, ...);
                ~~~~~~~~~~~~~~~~~~~~~^~~
drivers/net/phy/rtl8367b.c:729:51: warning: passing argument 2 of '_dev_info' makes pointer from integer without a cast [-Wint-conversion]
  dev_info("checking chip_num=0x%x ver=0x%x...\n", chip_num, chip_ver);
                                                   ^~~~~~~~
./include/linux/device.h:1346:51: note: in definition of macro 'dev_info'
 #define dev_info(dev, fmt, arg...) _dev_info(dev, fmt, ##arg)
                                                   ^~~
./include/linux/device.h:1295:54: note: expected 'const char *' but argument is of type 'u32' {aka 'unsigned int'}
 void _dev_info(const struct device *dev, const char *fmt, ...);
                                          ~~~~~~~~~~~~^~~
cc1: some warnings being treated as errors
```

Reply:
Argument `dev` was missing. Should be fixed now.